conans/cli/api/subapi/new.py

```diff
@@ -0,0 +1,81 @@
+import fnmatch
+import os
+
+from jinja2 import Template
+
+from conans.cli.api.helpers.new import new_templates
+from conans.cli.api.subapi import api_method
+from conans.cli.conan_app import ConanApp
+from conans.errors import ConanException
+from conans.util.files import load
+from conans import __version__
+
+
+class NewAPI:
+
+    _NOT_TEMPLATES = "not_templates"
+
+    def __init__(self, conan_api):
+        self.conan_api = conan_api
+
+    @api_method
+    def get_template_files(self, template):
+        app = ConanApp(self.conan_api.cache_folder)
+
+        # Find the template
+        template_files = None
+        if os.path.isdir(template):
+            template_files = self._get_files(template)
+        else:
+            folder_template = os.path.join(app.cache.cache_folder, "templates", "command/new",
+                                           template)
+            if os.path.exists(folder_template):
+                template_files = self._get_files(folder_template)
+
+        # If not in cache or user, then check if it is a predefined one
+        if template_files is None:
+            template_files = new_templates.get(template)
+
+        if not template_files:
+            raise ConanException("Template doesn't exist or not a folder: {}".format(template))
+
+        return template_files
+
+    def render(self, template_files, definitions):
+        excluded = template_files.pop(self._NOT_TEMPLATES, None)
```

Comment:
Checking here again the `_NOT_TEMPLATES` is a bit messy. Something is wrong.
What if these `get_xx_files` returns a tuple with (template_paths, files_paths) (instead of contents) and this `render()` only accept the template_paths? maybe the files_paths can just be copied.

Reply:
Yes, that makes sense. The problem is the asymmetry for built-in templates, so it is better if the method return the contents of the files already, but as long as the files are splitted into "files-to-render" and "files-to-not-render", that would be fine?